frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js

```diff
@@ -325,6 +325,17 @@ describe("scenarios > admin > settings", () => {
     cy.findByText(/Site URL/i);
   });
 
+  it("should display the order of the settings items consistently between OSS/EE versions (metabase#15441)", () => {
+    cy.visit("/admin/settings/setup");
+    cy.get(".AdminList .AdminList-item")
+      .as("settingsOptions")
+      .first()
+      .contains("Setup");
+    cy.get("@settingsOptions")
+      .last()
+      .contains("Whitelabel");
```

Comment:
That will fail on OSS. Whilelabel is an EE feature.

Reply:
Does it make sense to check for ALL members of the array and their position in UI?

Reply:
I'll fix this, btw.
Great catch - I totally overlooked it.

Reply:
No, keep it semi-flexible, but it's the second time that Whitelabel has ended up on top, so a test for first+last seems good.